leases/tests/test_lease_mutations.py

```diff
@@ -1395,3 +1400,184 @@ def test_create_winter_storage_lease_creates_contract(
     contract = lease.contract
 
     assert isinstance(contract, WinterStorageContract)
+
+
+TERMINATE_BERTH_LEASE_MUTATION = """
+mutation TERMINATE_BERTH_LEASE($input: TerminateBerthLeaseMutationInput!) {
+    terminateBerthLease(input: $input) {
+        berthLease {
+            status
+            endDate
```

Comment:
Hmmm it would make sense. My reasoning behind this was again for consistency with all the invoicing features where the date input is the `due` date; also to avoid confusion with the `end_date` field of the lease. But I think we can go with `end_date` to make clearer what it refers to.